dev-docs/bidders/admatic.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: bidder
+title: AdMatic
+description: Prebid AdMatica Bidder Adapter.
+pbjs: true
+biddercode: admatic
+media_types: banner,video
+---
+
+### Description
+
+One of the easiest way to gain access to AdMatic demand sources  - AdMatic header bidding adapter.
+
+AdMatic header bidding adapter connects with AdMatic demand sources to fetch bids for banner network ID. Please reach out to your account manager or <prebid@admatic.com.tr> for more information.
+
+### Bid params
+
+| Name         | Scope    | Description                        | Example    | Type     |
+|--------------|----------|------------------------------------|------------|----------|
+| `networkId`  | required | The network ID from AdMatic           | `12345`    | `number` |
+| `floor`| optional | Bid floor         | `0.5` | `float` |
```

Comment:
The table is not properly formatted. See https://deploy-preview-4118--prebid-docs-preview.netlify.app/dev-docs/bidders/admatic

I assume there are a few whitespaces too much. Or not enough :sweat_smile:

Reply:
Removing this is the easiest way :+1: